Atomic/atomic.py

```diff
@@ -326,12 +326,15 @@ def _pull_dockertar_layers(self, repo, imagebranch, temp_dir, input_layers):
     def _pull_image_to_ostree(self, repo, image, upgrade):
         if image.startswith("ostree:"):
             self._check_system_ostree_image(repo, image, upgrade)
-        elif self.args.image.startswith("docker:"):
-            self._pull_docker_image(repo, image.replace("docker:", ""))
+        elif self.args.image.startswith("oci:"):
+            self._check_system_oci_image(repo, image, upgrade)
         elif self.args.image.startswith("dockertar:"):
             self._pull_docker_tar(repo, image.replace("dockertar:", ""))
-        else: # Assume "oci:"
-            self._check_system_oci_image(repo, image, upgrade)
+        else: # Assume "docker:"
+            image = image.replace("docker:", "")
+            if ":" in image:
+                raise ValueError("Error: improper registry %s. Proper registries are docker, dockertar, oci, and ostree." % image)
+            self._pull_docker_image(repo, image)
 
```

Comment:
We were seeing issues here when trying to pull fedora, it was trying to pull it via ostree, I thought?

@mtohmaz PTAL